tensorflow-frameworks/tensorflow-data/src/main/java/org/tensorflow/data/Utils.java

```diff
@@ -0,0 +1,34 @@
+package org.tensorflow.data;
+
+import org.tensorflow.tools.Shape;
+
+public class Utils {

```

Comment:
It would be worth it to check which of the following utilities could be added directly to the `Shape` class instead. 

For example, `array(...)` could be replaced by something like `shape.extend(batchSize, 0)`, where 0 is the dimension index

Reply:
That makes sense; I removed this class, and added `Shape.head()` and `Shape.tail()` to the `Shape` class, and a new constructor for `Shape(int, int[])`.